Automattic/jetpack
# Adds social logos, adds telegram and whatapp sharebuttons.

#### Changes proposed in this Pull Request:
- Add new Social Logos icon font. The new version will be removing any sort of logos. 
- Adds the Telegram Social Logos
- Adds the WhatsApp Social logos.
- This is so that we can be in sync with .com code base. 

*To test*
- Add the new social logos, Telegram, WhatsApp Do they show up as expected. 
- Are you able to see them in all 4 version on the front end and in different browsers. 
- Are you able to make changes to the share buttons from calypso? 

When you activate the plugins compatible plugins does it break the site? 
- Activating https://wordpress.org/plugins/whatsapp-jetpack-button/ adds lets you add both whatsapp icons. 
- Activating https://wordpress.org/plugins/telegram-sharing-button-for-jetpack/ doesn't break the site. The user will have to deactivate the telegram sharing button for jetpack to have the best experience. This is because of the short name naming conflict. We can't update the shortname to resolve this conflict. 

cc: @jeherve, @lezama, @ebinnion 


-------------------
